start

```diff
@@ -181,6 +181,19 @@ load(
     "rules_haskell_toolchains",
 )
 
+load(
+    "@rules_haskell//haskell:cabal.bzl",
+    "stack_snapshot"
+)
+
+stack_snapshot(
+    name = "stackage",
+    packages = [
+        "time", # Provides the dependency @stackage//:time
```

Comment:
Redundant comment.

Reply:
Comment has been removed in c40197be5973b2e70bb0f7a33f95e388cbd25962.